airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java

```diff
@@ -44,9 +38,41 @@ public StandardSyncOutput run(final JobRunConfig jobRunConfig,
                                 final StandardSyncInput syncInput,
                                 final UUID connectionId) {
 
-    StandardSyncOutput syncOutput = replicationActivity.replicate(jobRunConfig, sourceLauncherConfig, destinationLauncherConfig, syncInput);
     final int version = Workflow.getVersion(VERSION_LABEL, Workflow.DEFAULT_VERSION, CURRENT_VERSION);
 
+    final ReplicationActivity replicationActivity;
+    final NormalizationActivity normalizationActivity;
+    final DbtTransformationActivity dbtTransformationActivity;
+    final PersistStateActivity persistActivity;
+
+    /**
+     * The current version calls a new activity to determine which Task Queue to use for other
+     * activities. The previous version doesn't call this new activity, and instead lets each activity
+     * inherit the workflow's Task Queue.
+     */
+    if (version > PREV_VERSION) {
```

Comment:
@evantahler @cgardens discussion from parent PR: 
![image](https://user-images.githubusercontent.com/7798112/186286380-33bd207a-0d71-4017-88b5-c5009c0ff602.png)

Reply:
I think we still need versioning, at least until https://github.com/airbytehq/airbyte/pull/14758 is merged.

Reply:
got it. let's use this versioning for now until we have confirmed we don't need it.

Reply:
Agreed!